index.js

```diff
@@ -50,6 +53,18 @@ module.exports = async (args, flags) => {
     await writeFile(`${flags.outputDir}/hubble-data.json`, prettyJSON(mapping));
   }
 
+  if (filesToDownload) {
+    const fullAssetsDir = `${flags.outputDir}/${ASSETS_DIR}/`;
```

Comment:
Ending paths with a trailing slash is generally considered less clean when you are concatenating them in an unsafe way:

```js
const path = '/usr/local/'
const binPath = `${path}/bin`
console.log(binPath) // /usr/local//bin
```